### PR TITLE
feat: Draft blog use cq for EventBridge crossaccount

### PR DESCRIPTION
**Pull request by jsonpr**
Adding blog for how a customer used CQ to find EventBridge Cross Account Usage.

### CODE REVIEW

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Finding Cross Account AWS EventBridge Usage
+tag: security
+date: 2022/11/25
+description: >-
+  How a CloudQuery customer wrote custom queries to find cross account AWS EventBridge Usage to help their teams migrate to new AWS EventBridge security features to increase security and compliance.
```
Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:6]: This is quite a long sentence with a lot of "to"s, can we maybe simplify / split it into multiple sentences? 😄

Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:6]:
```suggestion
  How a CloudQuery customer wrote custom queries to find cross-account AWS EventBridge Usage to help their teams migrate to new AWS EventBridge security features to increase security and compliance.
```

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+author: jsonkao
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Overview
+
+Recently, AWS sent out customer notification emails regarding upcoming changes for EventBridge cross account event bus targets.  In short, Amazon EventBridge today does not require a IAM role when sending events to a cross account event bus target.
```
Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:16]:
```suggestion
Recently, AWS sent out customer notification emails regarding upcoming changes for EventBridge cross-account event bus targets.  In short, Amazon EventBridge today does not require an IAM role when sending events to a cross-account event bus target.  
```

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+<BlogHeader/>
+
+## Overview
+
+Recently, AWS sent out customer notification emails regarding upcoming changes for EventBridge cross account event bus targets.  In short, Amazon EventBridge today does not require a IAM role when sending events to a cross account event bus target.  
+
+## EventBridge Change
+
+Beginning February 16, 2023, Amazon EventBridge will start requiring IAM roles for all new cross account event bus targets.
```
Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:20]:
```suggestion
Beginning February 16, 2023, Amazon EventBridge will start requiring IAM roles for all new cross-account event bus targets.
```

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+
+## Customer Query
+
+We would like to thank `jbarney` for sharing and writing the below query.  We're especially happy when our users bring innovation and layer on advanced queries on top of CloudQuery data to provide value to their organizations.
```
Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:25]:
```suggestion
We would like to thank `jbarney` for sharing and writing the below query.  We're especially happy when our users bring innovation and layer advanced queries on top of CloudQuery data to provide value to their organizations.
```

Comment (hermanschaaf) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:25]: Let's also add a short explanation of what this query does here

Comment (jsonpr) [reply website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:25]: Good idea!

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md (hunk 1 above)
Comment (jsonpr) [reply website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md]: Split, thanks!

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+(
+	SELECT account_id, name, policy, arn,
+	  regexp_matches(policy, '[0-9]{12}:root', 'g') as ext_account
+	FROM aws_eventbridge_event_buses
+) data
+WHERE account_id != ext_account[1];
+```
+The above query will detect any usage the AWS account reference for cross-account access to Amazon EventBridge Event Buses and will return a table of each occurrence of a cross-account reference. If there are multiple accounts referenced in a policy, each account will be a separate row.
+
+By filtering on the regex `[0-9]{12}:root`, we look for any string that matches part of an AWS account resource identifier such as `1213412341234:root`.  While we do look through the entire policy, AWS Account arns should only exist in the `Principal` block of statements.
```
Comment (disq) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:67]:
```suggestion
By filtering on the regex `[0-9]{12}:root`, we look for any string that matches part of an AWS account resource identifier such as `1213412341234:root`.  While we do look through the entire policy, AWS account ARNs should only exist in the `Principal` block of statements. 
```

**Review by yevgenypats — APPROVED** [review]

File: website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## References and Useful Links
+
+[AWS: Service Control Policies (SCPs)](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_scps.html)
+
+[AWS: Sending and receiving Amazon EventBridge events between AWS Account](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cross-account.html)
+
+[AWS EventBridge API Reference: PutTargets](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutTargets.html)
+
+[CloudQuery: AWS Plugin](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/aws)
```
Comment (yevgenypats) [inline website/pages/blog/how-to-find-cross-account-aws-eventbridge-usage.md:81]: Docs are now here: https://www.cloudquery.io/docs/plugins/sources/aws/overview

**Review by hermanschaaf — APPROVED** [review]
🌟

**Pull request merged**